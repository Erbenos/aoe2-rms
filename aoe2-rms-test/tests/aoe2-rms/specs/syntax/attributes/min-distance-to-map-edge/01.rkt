#lang aoe2-rms
; Ensure that relics stay at least 10 tiles from the edge of the map.

<OBJECTS-GENERATION>
(create-object 'RELIC
               (set-gaia-object-only)
               (number-of-objects 500)
               (min-distance-to-map-edge 10))
