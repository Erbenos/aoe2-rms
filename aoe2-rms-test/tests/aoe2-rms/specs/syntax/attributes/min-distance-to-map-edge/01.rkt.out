<OBJECTS_GENERATION>
create_object RELIC {
  set_gaia_object_only
  number_of_objects 500
  min_distance_to_map_edge 10
}
